definitions/procedures/foreman/fix_corrupted_roles.rb

```diff
@@ -2,7 +2,7 @@ module Procedures::Foreman
   class FixCorruptedRoles < ForemanMaintain::Procedure
     metadata do
       for_feature :foreman_database
-      tags :pre_migration
+      tags :pre_migrations
```

Comment:
As respective checks(`CheckCorruptedRoles` and `InvalidReportAssociations`) with tag  `pre-upgrade` are available, I don't think we will need to apply tag `pre_migrations` to these procedures (`FixCorruptedRoles` and `InvalidReportAssociations`).

Could you please remove these tags?

Reply:
removed `pre_migration` tag.